app/models.py

```diff
@@ -64,14 +73,30 @@ def check_password(self, password):
     db.Model.metadata,
     db.Column('user_id', UUID(as_uuid=True), db.ForeignKey('users.id')),
     db.Column('service_id', UUID(as_uuid=True), db.ForeignKey('services.id')),
-
     UniqueConstraint('user_id', 'service_id', name='uix_user_to_service')
 
 )
 
 
-class Service(db.Model):
+class ServiceHistory(db.Model):
```

Comment:
Why can’t this inherit from the `Service` model and override the things that are different?